### ISSUE TITLE
"sendTemperatureOnlyWhenOff" can't work

### ISSUE THREAD

**Issue by phenix13** [lprhodes/homebridge-broadlink-rm#313]
When set it true or false , the air-conditioner always turned on when setting a temperature while the air-conditioner is off.

**Comment by tamasvicsotka** [comment]
same for me

**Comment by lprhodes** [comment]
This should be fixed in the latest release

**Comment by lprhodes** [comment]
Actually I think I mis-understood the issue, taking another look.

**Comment by lprhodes** [comment]
OK, let me know how you go in version 3.2.20

**Comment by tamasvicsotka** [comment]
same result

Mar 20 10:33:38 pimaster homebridge[7004]: [3/20/2018, 10:33:38 AM] [Broadlink RM] A/C setCurrentHeatingCoolingState: 0
Mar 20 10:33:38 pimaster homebridge[7004]: [3/20/2018, 10:33:38 AM] [Broadlink RM] A/C setCurrentHeatingCoolingState: already 0 (no data sent - B)
Mar 20 10:33:39 pimaster homebridge[7004]: [3/20/2018, 10:33:39 AM] [Broadlink RM] A/C sendTemperature (25
Mar 20 10:33:39 pimaster homebridge[7004]: [3/20/2018, 10:33:39 AM] [Broadlink RM] A/C sendHex (192.168.1.57; 34:ea:34:f4:20:e9) 2600ca00919012341213113512351211121311351212121111361212111311351137111212351235111212361234123512351235113612121136121112121112121211131211123512351212111212351235121212111212111211361236111212111136123612ab8f9012351212113612351113111211361211111312351212121112351235111311351137111212351234123612351135113712111235121112121212121211121112133412361211121211351236111212121211121211361235121112121136123512000d050000000000000000000000000000

**Comment by lprhodes** [comment]
@tamasvicsotka The reason for `sendTemperatureOnlyWhenOff` is `When set, this will prevent the air-conditioner from being turned when setting a temperature while the air-conditioner is off.`

The log you sent doesn't suggest that that isn't working. I could be missing something though.

**Comment by lprhodes** [comment]
Or to word is another way... `sendTemperatureOnlyWhenOff` will only send the temperature when off, and not update the air-conditioner mode to heat/cool.

**Comment by lprhodes** [comment]
@tamasvicsotka I've added an option named `ignoreTemperatureWhenOff` in 3.3.4  in case that's what you're after.

**Comment by phenix13** [comment]
@lprhodes I mean that I change the temperature and the broadlink will not  send hex and update the AC mode to heat/cool when the AC is off . the only way you can to  make AC on is change the AC mode to heat/cool then change the temperature. because my temperature code has the information with AC on, so when send temperature hex will case my AC on. what can i do to use "ignoreTemperatureWhenOff" in my config

**Comment by lprhodes** [comment]
OK, that's the opposite of the issue you raised. No problem, I'll check it out

**Comment by lprhodes** [comment]
I assume you don't have `sendTemperatureOnlyWhenOff` set or have it set to false?

**Comment by lprhodes** [comment]
I'm not able to reproduce the issue @phenix13 

Can you please send your config

**Comment by tamasvicsotka** [comment]
Now it's perfect, thank you!

**Comment by phenix13** [comment]
{
				"name": "客厅空调",
				"type": "air-conditioner",
				"host": "192.168.1.7",
				"defaultCoolTemperature": 23,
				"defaultHeatTemperature": 26,
				"sendTemperatureOnlyWhenOff": true,
				"ignoreTemperatureWhenOff": true,
				"data": {
					"off": "260028010a000315000128951338151215121214153615351314131414131535131512381314121415131115151212141512121514131138131413141413121414141115143615121536121514131138141511000292131512141513111514361512151213131512121514131115141311391215121415121215141311151413111515121214151212151413111514131115151112391300052400011f9513381512141311151535153613151115141311381315123813141215131411151413111515121214141312381315111515121115141312141536153612391412151212381414110002931314121513141115141312141413121414131215131411151314111514131214141312151314111513371536131412151314111513141115141311151413123813000d05",
					"temperature30": {
						"pseudo-mode": "heat",
						"data": "260028010001299317361315131412381513111512151115131412381437123815131214131412141414111514131115141312381414111514131115121512141338121513371214151311391413120002921513121414131214133812151215111512151313131412151314113815131214141312141314121414131215121511151413121414131214131412141437121514000522000121931536131414131238151311151215121412141437143712381513131313141214131412151413111512151337131413131413121514121413143613381437121514131138151312000292151311151215121414131215121513131413111514131214141312141314141215121215141212151413121414121314131414131413131314361314141312141300096d0a000d05"
					},
					"temperatur29": {
						"pseudo-mode": "heat",
						"data": "26002401000129951338151214131139131412151413111513371512143715351314121514131115151212151215111514131139141312151413111514131114163515121436151214131138131412000293151212141413111515361412121511141413121513141114151212381314121414121215141311151314111513131214141311141513111513371535131412151500052100013b9418351415141516361416151313161513133916131339153715141612131411151314111415141712141516351314131514161513131516131339163615381712141417351a11160002931a1117131a1117131a1116141a111615191216141a1116141a1116141a1116141a111615191316111a121314151417121415171114151711151517111514171115000d0500000000"
					},
					"temperature28": {
						"pseudo-mode": "heat",
						"data": "2600240100011394143613131412123715111312151212131412121315351337141213121511131314121213141212131511133712141213151114111511141214361213153513131412133615111300029112141213151113131436141115111213151213121412121314121237151212131511141115111313141212131511131314121213151112131511133713131213150005210001139314371312151113371412141115111312151213121436123715111313141213121511131215111412141213361313131314121312141212131535143614361411151113371412130002911412131215111312151114121412131214121213151114121412121314121312151113131313131214121213151114111511141214121312153513371436123715000d0500000000"
					},
					"temperature27": {
						"pseudo-mode": "heat",
						"data": "2600240100012493133913131314133713141511131513131338133713141536131414121314141213141313131414131215133714131412131414121215131312391313123913131214143712141500028e151414121214151113381511131513121314141312141412121415351314141213141511131414121314141217121513171216121712161217361513171215131700052200012f92183515131712153717121513171215131736173515141537181115131712151317121513151416121811153715141513171215131514151317361537173615131712163617121900028f1a111a101b1015131b1017141a1117131a11171319121812191215141811151315141613151218121a1119111b1117131912181219121812191218361936183717000d0500000000"
					},
					"temperature26": {
						"pseudo-mode": "heat",
						"data": "2600240100012b91163615121512133817111611151215121612143716121238161213141512131416121412161215121512153615131412151215131413141215361513143715111411133514121400029016121412151314121536151314121512151215111612141215131436151215121512151314131412151215111612141215121513141215131412151215131411160005220001079315361512151314361512151215121512151214361612143616121412151314121512151314121512151215361512141314131412151215121536153615371412151215361512150002901812161217121414161417111712171215121512161215121513131418121612181216131712171217121711181217121712171217121612183615141736183517000d0500000000"
					},
					"temperature25": {
						"pseudo-mode": "cool",
						"data": "2600240100012592123810141214133611141213111413121138131513151238131512151513121514131215151212151315123813131312111412131214121312381114113814111214123713131300029013131213111513111238111412141213111412131214111411141336121412131214121311141213121412131114121312141213111412131238113811381237110005240001069511381213111412361214111311141114113811141114123612141113111412131114121212141014111412371114111411141212111511131138113811381114111412371114120002911114121212141014111412131114121311141113111412131114111411141113111412131114121311141113111411141114121311141113111412131138123711000d0500000000"
					},
					"temperature24": {
						"pseudo-mode": "cool",
						"data": "2600240100012a92153615121512143614131413151213131512141215121436151214131413151316131312161213131512143617131612161415131613161216361713163617121713163518121600029017121712141313131535151116121412151214121512151316131635171316131613161216131613161316121613161216141513161316121613143617361735180005220001409218351713161316351813151316131612161316131613163517131613161316121613161316131612161316361712171216131613161316121636173617361713161316351713140002901512141315121312171316121613161216141513161316121613161316131612161316131613161216131612171316121613161216141513163617351813163517000d0500000000"
					},
					"temperature23": {
						"pseudo-mode": "cool",
						"data": "2600240100012595143815121414133713151215151212151536153613381612151212151512121515121215151212141612123813151215151212151513111515361413163516121512123814151100029313141315151212151536151216121214161212151512121515121238131412151513121416121215151212151512121515121215151212151536151215371535130005250001269414381512161212381315121515121215153615361338151215121214151311151413121416121215151212381414121515131115151312141635163513391512151311391315120002921414121516121116151212151512121515121215151212151512121515131115161212141612121515121215151212151512121515131115151311391315123813000d0500000000"
					},
					"temperature22": {
						"pseudo-mode": "cool",
						"data": "2600240100012994133914131315123813151215131513131315123814381413131512151314121513151314121512151215133713151215121513141215131412381414123814141215123813151300029112151115131412151239131413141314121512151215121413151336141513131314121612141215121513141314131313151313131511151314121512381437140005230001269311381114111411381315131313141014111411381138111411141114111410141114111411141114111410381214101411141114131511151338143713381413131513361414120002931712171314141712151417121414171215151612151417121514171314141712151417131414171214151512131513131115131311151413133817121514173614000d0500000000"
					},
					"temperature21": {
						"pseudo-mode": "cool",
						"data": "2600240100013095133816121315113913151413131415121437151313381512131512151215141313151214131514131215133714151413121513141315121413381414123811141114113811141100029213151413131412151238151312151313131514121315111513141238131512151215131412151313131511151315111512151214131513141238133813151238130005240001249414381413131413371315121512151215123814141238141412151214131512141314121513141215121512381315131313151115131513131338173616371314151418351314150002921713171116141711131513141314141313151314141417121215131413151015131511151215121513141114131413151215141313151313131513141315133713000d0500000000"
					},
					"temperature20": {
						"pseudo-mode": "cool",
						"data": "2600240100011d951239111412141138121411141214111511141114123812141214111412141114121411141214111412141138121411141214111412141114123811151138111412141138121410000293111411141215101412381114111411141214111412141114111510381313111411141114111510141114111411141113121411141213111412141138111411381300052200010c9513371114111411381313111411141114121411141138121312141114121411141114111412141114111411391213101413121114131213141238123813381314131511381415110002931712171215141613151417121514171215151215151317111614161216141612161417111614161216141711161417111615111416141612163814391637151415000d0500000000"
					},
					"temperature19": {
						"pseudo-mode": "cool",
						"data": "26002401000128951339121513151238141411151413121414371238151311151215111512151214141312141315111514131138151312141413121512151115143613141338121512141238141312000292141411151215111513371215121412141413121412151214121511381513111512151115121412141215131314131115121511151215111513371214131412381400052200011e9315361214131413371413121413141214143612381513131314131115121511141314121412151115121513371413111513131214131413131436133814371214121513371414160002911514161216141513151416141414151315141614141415131514151415141414161415131514161215151414151416121515151315141613151414381638141415000d0500000000"
					},
					"temperature18": {
						"pseudo-mode": "cool",
						"data": "2600240100013195163614141214103812141114131212151413133815131215141312151512131514131413151312141513133815131215141312141613121514361314153713141512133815131200029216131115131512141437131315141214141313141414111415131436151312151314121514131215121413141315121414141115141311141313101413131038150005220001249415371215141312381513121512151215141312381513121413151115131412151314121514131215141312381513121414141115111411131337103813371014131310381312120002931712161315141514141416131514161315141613121512151314121513151115151312151215121513151215121512141414131413151215123913141338151415000d0500000000"
					},
					"temperature17": {
						"pseudo-mode": "cool",
						"data": "2600240100014394153914141515163617131611151417121538141415141712151417111514171215141711151417121514173518111712151417111515161215381711153914151414173518121700028f15151712141417111539131514121215141311151413111515121238131412151413111514131115151212141512121514131115161217121438173615381711150005250001409213391513151417351515171214151612133815121512121414151612151516121415161313151712141516361514171214151612141616121438173618351513151616351613170002901811171215141711161416121514171115151612151317131514161214141811181118121a1118121a111912191218121a111812191317111a1119121936161215000d0500000000"
					},
					"temperature16": {
						"pseudo-mode": "cool",
						"data": "2600240100012a941438131415131138131412151512121415121215141311151512121415121214151212151413111515121138131512141512131414131115153515121536121514131138131412000292131413131612121414361214151311141512121415121115151211381314131415121213151312141412131315121213161212131513111415121337133814121500052200011c9413381512141312371513121315121313151212131612111415131214141213131512111416121213151312371414111415121313151212141535153513381314151212381214130002921713161215141613151317121513181215131713151317121513171315131712151317131513141315141712161316121513171315131712153816361613171315000d0500000000"
					}
				}
			}

**Comment by phenix13** [comment]
@lprhodes I set both "sendTemperatureOnlyWhenOff" and "ignoreTemperatureWhenOff" to true, now I change the temperature the broadlink don't sent any hex , but there is another question , I change the mode to heat/cool to make the AC on , but broadlink don't send any hex . then I change temperature , it still don't sent any hex , only when i turn it off , it send a hex

**Comment by phenix13** [comment]
@lprhodes 
"name": "客厅空调",
				"type": "air-conditioner",
				"defaultCoolTemperature": 23,
				"defaultHeatTemperature": 26,
				"sendTemperatureOnlyWhenOff": true,
				"ignoreTemperatureWhenOff": true,

**Comment by tamasvicsotka** [comment]
Yes, same for me. It's not turning on.

**Comment by lprhodes** [comment]
`sendTemperatureOnlyWhenOff` should be false not true (or not included at all)

**Comment by tamasvicsotka** [comment]
It's not included.

{
          "name":"A/C",
          "type":"air-conditioner",
          "maxTemperature": 30,
          "defaultCoolTemperature": 21,
          "defaultHeatTemperature": 25,
          "autoCoolTemperature": 27,
          "autoHeatTemperature": 14,
          "ignoreTemperatureWhenOff":true,
          "autoSwitch": "A/C Auto Switch",
                "data":{
            "off":"2600ca009190123412121136113612111212113612121112123512121113123412351212113612111136113611361136121112361234113612121113111211131135121212$
            "temperature30":{
              "pseudo-mode":"heat",
              "data":"2600ca009190123412121136123512111212123511131112113612121113113511361113113612351112123611351136113611361235121211361112111311121113111$
            },
            "temperature29":{
....

**Comment by tamasvicsotka** [comment]
If I set ignoreTemperatureWhenOff to false, than we are in the starting point of the problem. The temperature hex values contains the on command for the A/C. For example If I leave home, I have a scenario for it, which contains the A/C to turn off (and a temperature), but it turns on the A/C.

If I set the ignoreTemperatureWhenOff to true, than it nothing to send except the off hex code, even if I set it to heat or cool and change the temperature, so I can't turn on the A/C.

I think @phenix13 has the same problem

**Comment by lprhodes** [comment]
Just curious - why do you send the temperature and turn it off when you leave home?

**Comment by lprhodes** [comment]
(rather than just turning it off)

**Comment by tamasvicsotka** [comment]
Can I set the scenario without the temperature in the home app? I didn't find that option. If I add the A/C for a scenario in the home app, than there is the mode and the temperature value too.

**Comment by tamasvicsotka** [comment]
http://www.youtube.com/watch?v=nz696IgC0Gc

**Comment by lprhodes** [comment]
Ok I’ll do some testing with the scenarios tomorrow - thanks @tamasvicsotka 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by oiva** [comment]
I think I have the same problem with @tamasvicsotka (a bit hard to follow :). I have a scene in Home app "turn off the AC" which sets the mode to off and the temperature to 21ºC. There is no option to clear the temperature so it's always sent with the 'off' command.

The logs I see:
```
GMT HAPServer [CC:22:3D:E3:CE:30] HAP Request: PUT /characteristics
GMT Accessory [Homebridge] Processing characteristic set: [{"aid":6,"iid":11,"value":0},{"aid":6,"iid":13,"value":21}]
GMT Accessory [Homebridge] Setting Characteristic "Target Heating Cooling State" to value 0
[Broadlink RM] ILP setTargetHeatingCoolingState: 0
[Broadlink RM] ILP sendHex (10.0.0.10; 34:ea:34:40:8e:b1)
GMT Accessory [Homebridge] Setting Characteristic "Target Temperature" to value 21
[Broadlink RM] ILP setTargetTemperature: 21
[Broadlink RM] ILP Potential sendTemperature (21)
[Broadlink RM] ILP sendTemperature (21, heat)
[Broadlink RM] ILP setTargetTemperature: 21
[Broadlink RM] ILP setTargetTemperature: already 21
[Broadlink RM] ILP sendHex (10.0.0.10; 34:ea:34:40:8e:b1)
GMT EventedHTTPServer [::ffff:10.0.0.8] HTTP Response is finished
[Broadlink RM] ILP setCurrentHeatingCoolingState: 0
[Broadlink RM] ILP setTargetHeatingCoolingState: 0
[Broadlink RM] ILP setTargetHeatingCoolingState: already 0
```

So it first turns off the AC, then sets the temperature to 21ºC which turns the AC back on, and finally refuses to turn the AC off because it's already off (in homebridge).

I'll try the `ignoreTemperatureWhenOff` option and see if that helps. Thanks!

**Comment by oiva** [comment]
BTW this seems to only happen when the scene is activated by an automation rule. When the scene is triggered manually it only sends the 'turn off' command. Weird.

**Comment by phenix13** [comment]
it's not send any hex when the remote control is off, it's only send the hex when it's on , So i use broadlink only can learn code with temperature and on information , for this reason, everytime the broadlink send  temperature date will turn on my AC. If I know only temperature date I can set "sendOnWhenOff" to control it. The function I need is when I change the temperature the AC don't change mode to heat/cool while it is off . When I change mode to heat/cool the broadlink send a temperature date case the AC on , then I change the temperature, broadlink send temperature everytime. In short, send temperature date only when the AC is on, and change temperature don't change UI and send date while the AC is off

**Comment by lprhodes** [comment]
@tamasvicsotka 

I'm seeing what you expect to see with the following config:

```
{
        "name":"A/C",
        "type":"air-conditioner",
        "maxTemperature": 30,
        "defaultCoolTemperature": 16,
        "defaultHeatTemperature": 30,
        "ignoreTemperatureWhenOff": true,
        "data":{
          "off":"OFF_HEX",
          "temperature30":{
            "pseudo-mode":"heat",
            "data":"TEMP_30"
          },
          "temperature16":{
            "pseudo-mode":"cool",
            "data":"TEMP_16"
          }
        }
      }
```

1. Setup a test scene to turn off the AC with a temperature of 16
2. Set the mode to heat and the temperature to 25
3. Press the test scene
4. The A/C shows as off
5. OFF_HEX is sent
6. Temperature is set to 16
7. No temperature hex code is sent

**Comment by lprhodes** [comment]
@phenix13 

That's exactly what `"ignoreTemperatureWhenOff": true` does

**Comment by phenix13** [comment]
@lprhodes I has been tested  "ignoreTemperatureWhenOff": true , Now it's perfect, thank you!

**Comment by phenix13** [comment]
@lprhodes How about the different between "ignoreTemperatureWhenOff" and "sendTemperatureOnlyWhenOff" , Thank you again !!

**Comment by tamasvicsotka** [comment]
Thank you! Now it seems to me that it works perfect.

@lprhodes I have a question about the A/C accessory, but I don't want to make a new issue. My A/C can set in the heat mode from 17 to 30 celsius and in the cool mode 17 to 30 celsius too. If I add all the possibility, than only the cool hex code-s working. Do I something wrong? This is my config:


...
"autoSwitch": "A/C Auto Switch",
	        "data":{
            "off":"2600ca00919012341212113611361211121211361212111212351212111312341235121211361211113611361136113612111236123411361212111311121113113512121212113612341236121111131211111312111212121211121334123512351235113611ab909112341114113512351211121212351113111211361212111311351235121211361211123511371135123512121136113512361113111211121212123412131112113612351235121211121211111311131113111211131135123611351136113612000d050000000000000000000000000000",
            "temperature30":{
              "pseudo-mode":"heat",
              "data":"2600ca00919012341212113612351211121212351113111211361212111311351136111311361235111212361135113611361136123512121136111211131112111311131112113611121137113611351136121212121211113611131113111211121235123611ab909011361113123512341212121212351112111213351212121112351136121211351235121211361133143612351234123611121235121111131113111311121112133412121235123512351136111211131211113711121113121112121235113612000d050000000000000000000000000000"
            },
            "temperature29":{
              "pseudo-mode":"heat",
              "data":"2600ca008f9112351212123412351212111311351113111311361211111312341236121111361236111212351135123612341235123611121136111212121212121111131112113612121136121111361236121112111212123512121235111211121137123511ac8e9212341212113612351212111212351212111213341113111212361135121211361136121111361136113612351136113612111136121211131211121211121112133512121135111312351136121112121112123612111235121112131135123512000d050000000000000000000000000000"
            },
            "temperature28":{
              "pseudo-mode":"heat",
              "data":"2600ca00909012351113123412361211121212341212121211361211111311361235121111361236121112351235123512341137123511121235121212121211111212121112123612111113111211361136121211121113123512351136121112121136113612ab8f9112341113123611351212121111371112121113341212111311361234121211361235111211361235123511361235123511121137111212121112111312111212123512121211111211371235121111121312113611351136121211131234113612000d050000000000000000000000000000"
            },
            "temperature27":{
              "pseudo-mode":"heat",
              "data":"2600ca00909111351113123412361211111311351213111211361211121212351235121112351236121112351235123512341236123512111235121211131211121111131112123612111212123412361235111212111113113612351113111212121136123512ab8f9112341212123611351212121112361112121113341212111311361135121211361235121112361135123511361235123512111137121111131211111311121212123512121112123512351136121111131212123512341212121212121234123512000d050000000000000000000000000000"
            },
            "temperature26":{
              "pseudo-mode":"heat",
              "data":"2600ca00909112341212113612351112111312351212121112351112121311351235111211371235111211361235123512351235113612111137111211131112111211131113123511351311113712341235121211131211111212351113121212111136123512ab8f9112341213113612341113111311361211121113351113111211361234121311351235111212361135123512361234113612121136121111131211121212121112123512351212113611351236121111131211111312351212121112121135123611000d050000000000000000000000000000"
            },
            "temperature25":{
              "pseudo-mode":"heat",
              "data":"2600ca00919012341213113512351211121311351212121111361212111311351137111212351235111212361234123512351235113612121136121112121112121211131211123512351212111212351235121212111212111211361236111212111136123612ab8f9012351212113612351113111211361211111312351212121112351235111311351137111212351234123612351135113712111235121112121212121211121112133412361211121211351236111212121211121211361235121112121136123512000d050000000000000000000000000000"
            },
            "temperature24":{
              "pseudo-mode":"heat",
              "data":"2600ca00909112341113123511361211111311361113111211361112121211361235111211371235121111361136113612341236123511121137111212121211111212121112121311351212111212361135111311121137111212351235121112121136123512ab8f9112341113123611351113111211371211121113341212121212351234121211361235121112361135113611371135123512111236121111131211121211131112131111361211121211361235121112121136121211351236111212121135123512000d050000000000000000000000000000"
            },
            "temperature23":{
              "pseudo-mode":"heat",
              "data":"2600ca00919012341213113512351211121311351113121111361212121212341235111311361235121112361234123512351136113612121136121111131211111311131211111311351113123511361235121211121136111213351212121111121136123612ab8e9112351113113612341213111211361211111213351212111211361136111311351235121211361135113712351135123611121136111212121212111311121112131111361212113612341236121112121135121211361113111211131235123512000d050000000000000000000000000000"
            },
            "temperature22":{
              "pseudo-mode":"heat",
              "data":"2600ca00919012341212113612351211111312351212121112351212111312341235121211361235121111371135123512351136113612111236111212121112121112131112111311351136123612341235121212121135121212111213111212111136123512ab909012351113113612341212121212351211121113351113111212351235111311351136121211361234123611361135113612121136121111131112121311121211121212351235123512341236121112121234121311121113121111131135123612000d050000000000000000000000000000"
            },
            "temperature21":{
              "pseudo-mode":"heat",
              "data":"2600ca00909112341113123512351211121211361113111211361211121311351235111212361235121111361235123512351136123511121137111211131211111211131212111311351235111311361136111211131136111213111136121212111136113612ab909011351213113611351113121211361112121113341213111212351234121311351235121112361135113612361135123512121235121111131211121211131112131111361136121212341136121211131135121212111236121112121135113711000d050000000000000000000000000000"
            },
            "temperature20":{
              "pseudo-mode":"heat",
              "data":"2600ca00909112341212123512351112121211361113111211361211121311351235121112361136121112351235113612351136123512111236121111131112111212121113111311121136121112361234121211121236123413111136121212111235123512ab909011351212123612341212121211361211111213341213111211361135121311351136121112361234123512361135113612121136111211131112111312121112131112111136111312351235121112121235113612111236121111131234123612000d050000000000000000000000000000"
            },
            "temperature19":{
              "pseudo-mode":"heat",
              "data":"2600ca008f9212341113123511361211121212351113121111361211111311361235121111371136111211361235123511351236113611121136121211131112121112121211121311121136123411371136111211121137123512111212111212121235123512ab8f9112341113113711351212111211371211111212351212111311361234111311361235111211371135113611371135123512111137111212121112111311121212121211121136123512351136121112121235123511121212121212121234123512000d050000000000000000000000000000"
            },
            "temperature18":{
              "pseudo-mode":"heat",
              "data":"2600ca008f9111361113113512351211121311351212111211371112111312341236111212351235121211361234113612351136113612121136121111131112121211131112111311121113113612351135111312121235123413351113121111121136123612ab8f9011361113113612341213111211361211111311361113111211361235111311351236111211361234123612351234123612111136121112121212121211121211121212111213113511361235121211121235123512351212121111131136113612000d050000000000000000000000000000"
            },
            "temperature17":{
              "pseudo-mode":"heat",
              "data":"2600ca00909012351113123411361113111311351212111211371211111311351137111211361235121211361135113711351235113711121235111211131212111211131112111311121113111311361234121211131136123411361236111211121136123612ab8e9112351113123511361113121111361112111311361212111211361235121211351137111212351135123612351234123612111136121112131112111312111211131111131113111211361234121311121136113612351235121111131235123511000d050000000000000000000000000000"
            },

            "temperature30":{
              "pseudo-mode":"cool",
              "data":"2600ca00909111351113113512361211111312341212121211361211111312351235111211361235121211351137113612331336113511131135121311121113121112121112123612111235123412131112111311121113123512121211123511351236123412ab909112341113113611361112121211361113111212351112121212351235111212361136121112351136113612341236123513101136121211131112111212121112123611121136123512121112121211121113123512121112123512351136113612000d050000000000000000000000000000"
            },
            "temperature29":{
              "pseudo-mode":"cool",
              "data":"2600ca00909111351212113512361112121211351114111211361211121212351235121111361137121111361235123511351236113611121235111311131112121111131112113711121136121112121212111311121113113511131136123512341236113611ac8e9212341212123512351212111211361113111212351212111211371135111311361136121111361235123512351235123512111136121211131211111312111112133511131135111311121212111311121113123412121235123511361235123511000d050000000000000000000000000000"
            },
            "temperature28":{
              "pseudo-mode":"cool",
              "data":"2600ca00909112341113123512351112121211361113121111361212111311351235121211361235121112361135113612351136113612111137121111131112111211131212123512111113111212131112111311121211113711361234123512351136113611ac909012341213113611351113121211361112121112351213111211361234121311351136121112361135113611371234123511131136111211131112121212121112133411131112121211131112111311121212113612351234123612351135113712000d050000000000000000000000000000"
            },
            "temperature27":{
              "pseudo-mode":"cool",
              "data":"2600ca008f9111361113123411371211121212341113111311361211121212351136121111361236111212351135123611351235123611121136121111141112111211131211113711121113113512121112121311121212113512351212113611351236123512ab8f9111351212113611361212121111361212111212351212121111371135121211351236111211361136123512351235123511121136111312121211111311121112133512121112123512111113111311121212123412361211123511361235123512000d050000000000000000000000000000"
            },
            "temperature26":{
              "pseudo-mode":"cool",
              "data":"2600ca00909112341212123512351211121212351113111211361212111311351235121211361235121111371135113611361235113611121236121111131112121112121212123512341212123612111211111311121113121211361211123512351235113612ab909012341213113611351212121112361211121112351213111211361135121311351235121112361234123512361135123512121136111212121112121212121211133411361212113612111113121112121212111211361211113712351234123611000d050000000000000000000000000000"
            },
            "temperature25":{
              "pseudo-mode":"cool",
              "data":"2600ca00909112341212123512351211111311361212121111361113111312341235121211361235111212361234123512351235113612121136121112121112111212131112123511351312111212121112121211121212111311361234113711351235123512ab8f9112351113113611351113121211361112121113351113121111361136111311351235111311361234123512361234123512121235121111131211121311121211133412351212111311121112121211121213111212351234123611361135123611000d050000000000000000000000000000"
            },
            "temperature24":{
              "pseudo-mode":"cool",
              "data":"2600ca008f9112351212123412351212111312341212111212361211121212341137121112351235121211361234123611351136123512121235121111131211121211131211121212341213111212121211121212111236111212351234123612341235123611ac8f9012351212123512351113111211361211121211361212121112351235121212341236111212351135113712351234123612111136121112121212121212111211131112351212121112121112121212111236121112351235123512351235123512000d050000000000000000000000000000"
            },
            "temperature23":{
              "pseudo-mode":"cool",
              "data":"2600ca00909112341212113611361211121212351212111212351112111312351136111211371235121112351235123512351136113612111236121111131112121112121212121212341212123511131211111312111235121212351211123512351235123511ac8f9112341212123612341113121112361112121113341212121212351234121211361235121111371135123511371234123512111236121111131112121211131112121212351211113711121211111312111137121112351211113711361135123512000d050000000000000000000000000000"
            },
            "temperature22":{
              "pseudo-mode":"cool",
              "data":"2600ca00909012351113123411361212111311351113121111371112121211351236111211361235121112361234123512351136123512121136121111131211121212121112111311351236113611121112111312121136111211131112113711351136123611ac8e9112351212113611351213111212351112121113351212111212351235111311351235121212351234123612351135123612111235121111131113111311121211121211361235113612111113121211121334121212111212123512351235113612000d050000000000000000000000000000"
            },
            "temperature21":{
              "pseudo-mode":"cool",
              "data":"2600ca00909111351113123412361211111312341213111211361211111312351235111211361236111211361235123512341236113611121235121211131112111211131112121212351235111212121113111312111136121111131235123512351136123511ac8f9111351212113612351212121112351212111213341113111212361234121211361136121112351136123511361235123512111137111211131211121211121212121211361135111311131112121211121235111312121135113612361234123512000d050000000000000000000000000000"
            },
            "temperature20":{
              "pseudo-mode":"cool",
              "data":"2600ca00909012351113123411361212111311351113111212361112111312341137121111361136121212351234123612341235123611121136111211131212111212121211121212111236111211131211121211121236123412121136113611351235123611ac8e9112351113123511361212111211361112111312351212111211361235121211351137111212351235113611361234123612111136111212131112111312111211121211131136111212121211121211131136113512121235113611361136123512000d050000000000000000000000000000"
            },
            "temperature19":{
              "pseudo-mode":"cool",
              "data":"2600ca008f9013351212123411361212121211351212111211371211121211351137111212351235121211361234123612341235123611121136121111131212111211131211121211121236113511131112121212121136123412121212113612341235123612ab8f9012351212123512351113111211361211121211361212111211361235121211351236111212351234123612351234123611121235121111141112121212111211131111131136123511121211111411121235123512121112123512351235123512000d050000000000000000000000000000"
            },
            "temperature18":{
              "pseudo-mode":"cool",
              "data":"2600ca00909112341113123511361211111311361212121111361211121211361235121111371136121112351235113612341236123511121137111212121211121112121211121311121212123412121212111311121136123412361112113611361235123512ab909012341212113711351113121111371211121112351212111312351234121211361136111211371234113611361235113612111236111212121112111312111212121211121113123412131112111312111136113612351112123512361234123512000d050000000000000000000000000000"
            },
            "temperature17":{
              "pseudo-mode":"cool",
              "data":"2600ca00919012341212123512351211111312351212121111361212121211351235111312351235121111371234123512351136123512121136111211131211121211131112111311121113121111131212111212121234113711361135123611351235113711ab909012351212113612341212121211361211111213351113111211361235121212341235121212351234113711361234113711121136111212121113111311121112121211121212111311131112111312111136113612351234123612351234123611000d050000000000000000000000000000"
            }
          }

**Comment by lprhodes** [comment]
@tamasvicsotka please do create a new issue. It makes issues easier for others to find and resolve and my life easier in terms of keeping track of things.
